src/components/common/Input.tsx

```diff
@@ -2,6 +2,8 @@ import React, { InputHTMLAttributes } from 'react';
 import clsx from 'clsx'; // clsx 임포트
 import { twMerge } from 'tailwind-merge';
 import { FieldErrors, FieldValues, UseFormRegisterReturn } from 'react-hook-form';
+// Font Awesome 아이콘을 사용합니다.
+import { LuEye, LuEyeClosed } from 'react-icons/lu';
```

Comment:
여기에서는 원래 이름을 통해서도 어떤 아이콘인지 명확히 잘 알 수있긴 한데,
이런식으로 icon 이름을 변경하는 게 더 명확하게 잘 보이는 듯 합니다 :)
```suggestion
import { LuEye as IconEyeOpen, LuEyeClosed as IconEyeClosed } from 'react-icons/lu';
```

그리고 icon을 Map으로 이런식으로 관리해주면
```tsx
const passwordIconMap = {
  true: IconEyeOpen,
  false: IconEyeClose
}

const IconEye = passwordIconMap[`${showPassword}`]

```

조건부 렌더링을 하지 않아도 되겠네요!

before
```tsx
{showPassword ? (
  // ✨ ADDED: 눈 뜬 아이콘
  <LuEye size={20} color='#888888' />
) : (
  // ✨ ADDED: 눈 감은 아이콘
  <LuEyeClosed size={20} color='#888888' />
)}
```

after
```tsx
<IconEye size={20} color='#888888' />
```

Reply:
오 좋은 방법이네요!